From 1bb58b103ee2180818688a4ecf1aae40b470f4dc Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Fri, 9 Sep 2022 18:33:31 +0200
Subject: [PATCH 213/841] fix(st-ddr): phy: adjust phyvref for DDR4

Set Vref at 73.8% to increase efficiency.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 330e10f63f4cd16f60dce0cfb7ccd27725fe8455 fix(st-ddr): phy: adjust phyvref for DDR4]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: I4dd8d9a4ce7e30bb9231fbb91fde2bb4e5b3c124
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/265789
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c b/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c
index 72179a9b3..91865383b 100644
--- a/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c
+++ b/drivers/st/ddr/phy/phyinit/src/ddrphy_phyinit_initstruct.c
@@ -80,7 +80,7 @@ void ddrphy_phyinit_initstruct(void)
 					 * see PHY databook
 					 */
 #else
-	uint8_t phyvref = 0x56U;	/*
+	uint8_t phyvref = 0x5EU;	/*
 					 * Use Analytical VREF and Compensate for T28 Attenuator,
 					 * see PHY databook
 					 */
-- 
2.47.2

